setup.py

```diff
@@ -69,7 +69,13 @@ def parse_reqs(req_path="./requirements.txt"):
     zip_safe=False,
     install_requires=REQUIRED,
     include_package_data=True,
-    extras_require=dict(coverage=["chanjo-report"]),
+    extras_require={
+        "coverage": [
+            "chanjo@git+https://git@github.com/Clinical-Genomics/chanjo.git@4.6.1",
```

Comment:
I'd prefer having the temporary fix in chanjo-report and release a new version of that software instead of modifying this setup. Actually would be SO much better to have an updated chanjo in PyPI!

Reply:
Totally agree